brightlayer-ui/figma-plugins
# Add support for FigJam; Clean up

Fixes #23


#### Changes proposed in this Pull Request:
- Added support for FigJam
- Added changelog
- ~Addressed 2 dependabot PRs~ webpack v5 frustrated me; wrote #44 and fingers crossed
- Verified that all functions work in both Figma and FigJam, light or dark theme.


#### To Test:
- Follow the [readme instructions](https://github.com/brightlayer-ui/figma-plugins/blob/dev/variant-switcher/README.md)
- To test light / dark theme: switch your system preference theme to light / dark. Figma's UI will update itself to reflect the change. The plugin should also reflect the change
